### PR TITLE
Add Role.sql

### PR DESCRIPTION
**Pull request by sittipong-hemloun**
I added Role.sql to create the directory, But I am not sure what the data type of role_id should be. please check it for my first PR.
thank

### CODE REVIEW

File: create/Role.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE Role(
+    role_id INT(4) NOT NULL AUTO_INCREMENT,
```
Comment (Qu1etboy) [inline create/Role.sql]: The primary key already implied `not null`, so there is no need to write it again.

**Comment by Qu1etboy** [comment]
Don't forget to mark it as done in README as well.

**Comment by sittipong-hemloun** [comment]
I marked it as done. 👻

**Comment by Qu1etboy** [comment]
Remove not null ด้วย มันไม่จําเป็นต้องมีใน primary key ตรง role_id

**Comment by sittipong-hemloun** [comment]
ได้ๆ ลืมม55

**Comment by Qu1etboy** [comment]
👍

**Pull request merged**